powerbi-docs/developer/visuals/license-api.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Licensing API
+description: This article discusses how to retrieve the Power BI visual's licenses and trigger the display of out of box license related notifications to the PBI user.
```

Comment:
the visual should not display its own licensing UX - use ours.

Reply:
the developer should consider non-supported envs or outage periods so as not to lock the user's visual. we should provide some guidance here IMO.